frameworks/elastic/tests/test_sanity.py

```diff
@@ -167,6 +167,7 @@ def test_custom_yaml_base64():
 
 @pytest.mark.sanity
 @pytest.mark.timeout(60 * 60)
+@pytest.mark.skip(reason='MESOS-9008: Mesos Fetcher fails to extract Kibana archive')
```

Comment:
Could we restrict this to the master builds?

Reply:
Good idea! https://github.com/mesosphere/dcos-commons/pull/2539/commits/9e8fc02d270418b1b495c26bf02d4cb392da8555